### PR TITLE
Bo sort

### PR DESCRIPTION
**Pull request by brucepang**
1. Modify sort function definition to take in a userData object which is retrieved from firebase. userData also has user's location (long,lat) which is passed with url. In the future, userData object will likely contain user's preference which will be taken into consideration for sorting.
2. Currently, the sort is based on remaining time and distance from user to event. The score assigned to each event range from 0~2, with 1 point from each sorting. The scoring mechanism definitely needs further discussion. 
3. Events without start time needs to be filtered out. @sahibgrover  Otherwise, it will affect the sorting algorithm.

### CODE REVIEW

**Review by ssgrover — APPROVED** [review]
Great! Ready to merge

File: service/sort.js [hunk 1]
```diff
@@ -54,6 +66,30 @@ var sort =  {
 			event['score'] += timeScore;
```
Comment (mavysavydav) [inline service/sort.js:66]: @brucepang is this timescore variable set at this point?

Comment (brucepang) [reply service/sort.js:66]: Thanks for catching that. I think the code works but the naming is bad. I copied part of code from the previous part and forgot the variable name. Will make a commit soon.

Comment (mavysavydav) [reply service/sort.js:66]: sg!

**Pull request merged**